packages/components/src/components/filter-dropdown/filter-dropdown.tsx

```diff
@@ -1,26 +1,38 @@
 import * as React from 'react';
-import PropTypes from 'prop-types';
 import { isMobile } from '@deriv/shared';
-import Dropdown from '../dropdown/dropdown.jsx';
+import Dropdown from '../dropdown/index.js';
 import SelectNative from '../select-native/select-native';
 
+type TFilterDropdown = {
+    dropdown_className: string;
+    dropdown_display_className: string;
+    filter_list: Array<any>;
```

Comment:
Can we use another type here instead of any? maybe a generic type if the list can be anything